### ISSUE TITLE
Use a CNN for Melanoma Classification

### ISSUE THREAD

**Issue by alif-munim** [alif-munim/r-tensorflow#20]
The maximum accuracy achieved for melanoma classification using a traditional fully connected neural network was ~75%. Create a model using a convolutional architecture to attain higher accuracy.
